Readme.md

```diff
@@ -3,45 +3,147 @@
+        });
 
+        settings.CallbackRouteValues = new { Controller = "Home", Action = "MasterGridViewPartial" };
+        settings.CustomBindingRouteValuesCollection.Add(
+            GridViewOperationType.Sorting,
+            new { Controller = "Home", Action = "MasterGridViewSortingAction" }
+        );
+        settings.CustomBindingRouteValuesCollection.Add(
+            GridViewOperationType.Paging,
+            new { Controller = "Home", Action = "MasterGridViewPagingAction" }
+        );
+        // ...
 ```
 
-<p>Finally, the resulting grid view model object is passed from the Controller to a particular grid's Partial View as a Model. For a detail grid, the master row key value is also passed to the Partial View as a ViewData object. In the Partial View, each grid binds to the Model via the <a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewExtension.BindToCustomData(DevExpress.Web.Mvc.GridViewModel)"><u>BindToCustomData</u></a> method.</p><p>Note that when implementing the grid's custom data binding, the <a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.DevExpressEditorsBinder"><u>DevExpressEditorsBinder</u></a> must be used instead of the default model binder to correctly transfer values from DevExpress editors back to the corresponding data model fields. In this code example, the DevExpressEditorsBinder is assigned to the ModelBinders.Binders.DefaultBinder property within the Global.asax file, thus overriding the default model binder.</p><p><strong>See Also:<br />
-</strong><a href="https://www.devexpress.com/Support/Center/p/E4394">How to implement a simple custom binding scenario for GridView</a></p>
+**Detail grid:**
+```razor
+@Html.DevExpress().GridView(
+    settings => {
+        settings.Name = "detailGrid" + ViewData["CustomerID"];
+        settings.SettingsDetail.MasterGridName = "masterGrid";
+        settings.KeyFieldName = "OrderID";
+
+        settings.CallbackRouteValues = new { Controller = "Home", Action = "DetailGridViewPartial", CustomerID = ViewData["CustomerID"] };
+        settings.CustomBindingRouteValuesCollection.Add(
+            GridViewOperationType.Sorting,
+            new { Controller = "Home", Action = "DetailGridViewSortingAction", CustomerID = ViewData["CustomerID"] }
+        );
+        settings.CustomBindingRouteValuesCollection.Add(
+            GridViewOperationType.Paging,
+            new { Controller = "Home", Action = "DetailGridViewPagingAction", CustomerID = ViewData["CustomerID"] }
+        );
+        // ...
+```
+
+### Controller
+
+Action methods update the [GridViewModel](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewModel) object with the information from the performed operation. The [ProcessCustomBinding](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewModel.ProcessCustomBinding.overloads) method delegates the binding implementation to specific model-layer methods pointed by the method's parameters.
```

Comment:
`pointed by the method's parameters` Can you explain what this means?

Reply:
```suggestion
Action methods update the [GridViewModel](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewModel) object with the information from the performed operation. The [ProcessCustomBinding](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewModel.ProcessCustomBinding.overloads) method delegates the binding implementation to specific model-layer methods specified by the Action method's parameters.
```

Reply:
Ok